src/Data/Cursor/AbstractCursor.php

```diff
@@ -132,6 +144,153 @@ protected function is_valid_offset_and_node() {
 		return true;
 	}
 
+	/**
+	 * Validates threshold field configuration. Validation failure results in a fatal
+	 * error because query execution is guaranteed to fail.
+	 *
+	 * @param array $field  Threshold configuration.
+	 * 
+	 * @throws \GraphQL\Error\InvariantViolation Invalid configuration format.
+	 * 
+	 * @return void
+	 */
+	protected function is_valid_threshold_field( $field ) {
```

Comment:
Can we rename this to `protected function validate_threshold_field( $field ) : void {}`? More self documenting (`is_*` usually implies a bool return).